packages/form/src/components/RadioGroupField/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { Meta } from '@storybook/react'
+import { Form, RadioGroupField } from '../..'
+import { mockErrors } from '../../../mocks'
+
+export default {
+  component: RadioGroupField,
+  decorators: [
+    ChildStory => (
+      <Form onRawSubmit={() => {}} errors={mockErrors}>
+        {ChildStory()}
+      </Form>
+    ),
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: 'A group of radios field',
+      },
+    },
+  },
+  title: 'Form/Components/Fields/RadioGroupField',
+} as Meta
+
+export { Playground } from './Playground.stories'
+export { Error } from './Error.stories'
+export { Helper } from './Helper.stories'
+export { Direction } from './Direction.stories'
```

Comment:
Can you add a story with required prop and a submit button following what I have done on checkbox: https://github.com/scaleway/ultraviolet/blob/a3aa6381b656cb1a535926b0eb8f839602b94b14/packages/form/src/components/CheckboxGroupField/__stories__/Required.stories.tsx

So we see that when nothing is checked the submit is disabled until a selection is made

Reply:
Done:

<img width="616" alt="Screenshot 2023-08-29 at 14 26 57" src="https://github.com/scaleway/ultraviolet/assets/43268759/ec90af14-d60e-4300-a1be-12ac980dd06d">